build.gradle

```diff
@@ -57,13 +57,13 @@ subprojects { project ->
     }
     project.tasks.withType(JavaCompile) {
         dependsOn(installGitHooks)
-        options.compilerArgs += [
-            "-Xlint:deprecation",
-            "-Xlint:rawtypes",
-            "-Xlint:unchecked",
-            "-Werror"
```

Comment:
Moving this since we can't compile with -Werror on JDK 8 anymore (due to warnings caused by jspecify annotations)